lib/modules/manager/sbt/extract.ts

```diff
@@ -32,10 +37,41 @@ interface Ctx {
   depType?: string;
   useScalaVersion?: boolean;
   variableName?: string;
+
+  sbtVersion?: string;
 }
 
 const scala = lang.createLang('scala');
 
+const sbtVersionMatch = q
+  .sym<Ctx>('sbt')
+  .op('.')
+  .sym('version')
+  .op('=')
+  .num((ctx, { value: sbtMajor }) => ({ ...ctx, sbtVersion: sbtMajor }))
+  .num((ctx, { value: sbtMinor }) => ({
+    ...ctx,
+    sbtVersion: (ctx.sbtVersion as string) + sbtMinor,
+  }))
+  .num((ctx, { value: sbtBugfix }) => ({
+    ...ctx,
+    sbtVersion: (ctx.sbtVersion as string) + sbtBugfix,
+  }))
+  .handler((ctx) => {
+    if (ctx.sbtVersion) {
+      const dep: PackageDependency = {
+        datasource: GithubTagsDatasource.id,
```

Comment:
shouldn't it use releases datasource instead?

Reply:
I think Github releases are sometimes not generated whereas the tag is always but open to discussion. In the case of sbt, look's like a Github release is always generated.